editor/log.py

```diff
@@ -103,8 +103,8 @@ def setroot(cls, root: Holder):
 def silence(*args): pass
 
 
-def print_announce(message, local, root):
```

Comment:
Should we just delete this if its not being used?

Reply:
Guess so. It was useful for debugging at one point, but I think the web interface is sufficiently stable now so this is no longer necessary.